marklogic-data-hub-central/ui/src/pages/Bootstrap.scss

```diff
@@ -1,33 +1,33 @@
 /* TODO Test view for React Bootstrap components, Bootstrap.scss TO BE REMOVED */
 
-.mosaic-container-bootstrap .mosaic-window-body {
-    padding: 20px;
-}
+// .mosaic-container-bootstrap .mosaic-window-body {
+//     padding: 20px;
+// }
```

Comment:
@iveyMarklogic  Consider not leaving unnecessary comments in the code

Reply:
@yubiselmarklogic i removed this coments